### ISSUE TITLE
Change pointer to scissors when hovering over an existing matching

### ISSUE THREAD

**Issue by ronichoudhury** [Kitware/candela#439]
h/t @zachmullen

**Comment by alex-r-bigelow** [comment]
I think at one point I had this, but then scrapped it for some reason (I think I just didn't have time to figure out the CSS). There should even be a scissors icon that I designed for this purpose in app/resonantlab/images directory, somewhere in the git history (it's long since been deleted... when I get a chance I can dig it up).

**Comment by zachmullen** [comment]
Unfortunately I don't think there's a native scissors cursor. The closest visual thing might be `cursor: not-allowed`, but that could be confusing since it also might lead users to believe the action is disallowed.

**Comment by zachmullen** [comment]
(Alternative is simply to use an image via `cursor: url(...)`.)

**Comment by alex-r-bigelow** [comment]
Yeah, I've definitely done the `cursor: url(...)` thing before—there's also something else with specifying the size and hot spot of the cursor, and I thought there were some cross-browser issues, but I don't remember what they were off the top of my head. I think at the time I was in a rush and had bigger fish to fry.
